src/main/java/com/synopsys/integration/alert/web/controller/SystemController.java

```diff
@@ -100,55 +89,22 @@ public SystemController(final SystemActions systemActions, final ContentConverte
     }
 
     @GetMapping(value = "/system/setup/initial")
-    public ResponseEntity<String> getInitialSystemSetup(final HttpServletRequest request) {
-        final Optional<ResponseEntity<String>> previousSetupResponse = checkForInitialSetup(request.getServletContext().getContextPath());
-        if (previousSetupResponse.isPresent()) {
-            return previousSetupResponse.get();
-        }
-
-        return responseFactory.createOkContentResponse(contentConverter.getJsonString(systemActions.getCurrentSystemSetup()));
+    @Deprecated
+    public ResponseEntity<String> getInitialSystemSetup() {
+        // FIXME: 6.0.0 remove this method
```

Comment:
I have this here to avoid breaking changes but I could remove the method.

Reply:
I'm all for removing this as I can't imagine anyone was using any of these endpoints besides us. But it would technically be breaking.